lib/typescript/class-names-to-type-definition.ts

```diff
@@ -79,6 +81,10 @@ export const classNamesToTypeDefinitions = (
           .filter(isValidName)
           .map(classNameToNamedTypeDefinition);
 
+        if (options.banner) {
```

Comment:
Can we write a test case for this line/logic? It looks like we currently only have a test for the `default` case.